### DISCUSSION TITLE
huawei honor 7x error ACK is invalid! ACK=55; Excepted=AA

### DISCUSSION THREAD

**Discussion by k-holeseeit** [kitsuned/PotatoNV#77] [Q&A]
Verifying images...
Uploading hisi65x_a...
- xloader
ACK is invalid! ACK=55; Excepted=AA

test point mode

**Comment by ChanceHarrison** [comment]
I am having the same issue.

Edit: The wrong firmware was selected by default, be sure to read the readme on the repo to make sure you have the correct bootloader selected in the drop down. This was the cause of my issue, at least.

Additionally, if you're looking for support, the Telegram group is likely to elicit a quicker response. https://t.me/potatonvsupport

@k-holeseeit Are you still trying / still having the issue?
